### PR TITLE
feat: add more crash scenarios to example app

### PR DESCRIPTION
**Pull request by ahmedAlaaInstabug**
## Description of the change

1. add new crash scenarios to example app

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

JIRA Ticket: [MOB-13634](https://instabug.atlassian.net/browse/MOB-13634)
### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests

### Code review

- [ ] This pull request has a descriptive title and information useful to a reviewer
- [ ] Issue from task tracker has a link to this pull request

[MOB-13634]: https://instabug.atlassian.net/browse/MOB-13634?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by mnoseir** [comment]
### Coverage Report
| Label | Coverage | Status |
| ----- | -------- | ------ |
| JavaScript | 96.2% | ✅ |
| Android | 47.2% | ✅ |
| iOS | 63.1% | ✅ |
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against b4a202b5d57b8a01249cc466ca2f98d9dcce7561
</p>

File: examples/default/src/components/PlatformListTie.tsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+import { Box, HStack, Pressable, Text } from 'native-base';
+import { Platform } from 'react-native';
+
+interface PlatformListTileProps extends PropsWithChildren {
+  title: string;
+  onPress?: () => void;
+  platform?: 'ios' | 'android';
+}
+
+export const PlatformListTile: React.FC<PlatformListTileProps> = ({
```
Comment (abdelhamid-f-nasser) [inline examples/default/src/components/PlatformListTie.tsx:12]: I think I missed this when creating the branch, but file name is `PlatformListTie` and Class is `PlatformListTile`, there's a missing L in Tile. 🙈 🙏

**Review by abdelhamid-f-nasser — CHANGES_REQUESTED** [review]
Great effort 👍 💪 , please check the file name change request (it's a typo I did in the original PR 🙏 🙈 ).

File: examples/default/src/components/PlatformListTie.tsx (hunk 1 above)
Comment (ahmedAlaaInstabug) [reply examples/default/src/components/PlatformListTie.tsx]: fixed,, done

**Review by abdelhamid-f-nasser — APPROVED** [review]

**Pull request merged**